src/main/features/core/tray.js

```diff
@@ -15,12 +15,14 @@ let audioDeviceMenu = [
 
 if (process.platform === 'darwin') {
   appIcon = new Tray(path.resolve(`${__dirname}/../../../assets/img/macTemplate.png`));
+} else if (WindowManager.getWindowManagerGDMName() === 'linux') {
```

Comment:
OK, havent thought enough about it, my fault. havent had the time to work with electron as much as i wanted.